entrypoint.sh

```diff
@@ -91,6 +91,12 @@ test_pkg() {
     if [[ "${TEST_PKG}" && "${TEST_PKG}" != "${pkg}" ]]; then
         return 0
     fi
+
+    # run go vet and capture the result for the package, but still execute the
+    # test suite anyway
+    show go vet "./${pkg}"
```

Comment:
I think this fails because the build constraint/tag for the version (mimic/luminous) is not passed.

Reply:
Just tested this, and it seems to be the case. Use `show go vet ${BUILD_TAGS} "./${pkg}"` and it passes with mimic builds.

Reply:
All set. Thanks!